.buildkite/al2_pipeline.yml

```diff
@@ -12,7 +12,15 @@
 # permissions and limitations under the License.
 
 steps:
-
+  # Add the step to better know the environment running on
+  - label: ':ec2: show environment'
```

Comment:
`uname -a` will  spit the kernel name, version and platform info. What is your intention here? Can we not infer that based on the host the tests are running on?